### DISCUSSION TITLE
How to use after install?

### DISCUSSION THREAD

**Discussion by Volcanicus** [ModelSurge/sd-webui-comfyui#212] [Q&A]
I installed it using the link but whenever I load webui, any other non-generating tab is simply broken.
When clicking on the comfyui tab, it says cannot connect.

I downloaded the zip from here, exported it to my webui extensions folder.
Then I copied my ComfyUI folder into this extensions folder.
When I run webui, it looks like it loads fine but it doesn't work as others have shown.

Am I missing something?

**Comment by Volcanicus** [comment]
I actually saw there was an issue similar to this with no resolution.

**Reply by PladsElsker** [reply]
I don't understand the issue, can you show a screenshot, or link to the mentioned unresolved issue? 

Have you tried clicking the "reload frontend" buttons in the txt2img/img2img accordions of the extension?

**Reply by Volcanicus** [reply]
This one: https://github.com/ModelSurge/sd-webui-comfyui/issues/184
I am getting the same result.
My WebUI is 1.6 instead of 1.7.
Here is what all non generating tabs look like:
![image](https://github.com/ModelSurge/sd-webui-comfyui/assets/35196249/dd9cae83-946d-4be1-be46-06b47151ebe8)

I have not tried reloading the front end but nothing seems to work at all correctly. The first generation will proceed and then the whole system won't respond.

**Reply by PladsElsker** [reply]
Ah, see #176. The extension is incompatible with the mov2mov extension. 

Mov2mov duplicates elemids, the problem arises from their repo.
